frontend/src/features/public-form/components/FormEndPage/components/EndPageBlock.tsx

```diff
@@ -40,6 +41,11 @@ export const EndPageBlock = ({
     },
   })
 
+  const submissionTimestamp = useMemo(
+    () => format(new Date(submissionData.timestamp), 'dd MMM yyyy, HH:mm:ss z'),
```

Comment:
Since the type of `submissionData.timestamp` is already a `Date`, the `Date()` constructor wrapper here is unnecessary, correct?

Btw, I'm thinking the oversight is probably due to the change of types between Dto and app data type. In the PR so far, `SubmissionResponseDto.timestamp` is of type number, but `submissionData.timestamp` is of type `Date`.

To me, the term `timestamp` connotates a number (ms from epoch), so I'm thinking  maybe we could avoid some confusion with some slight refactoring. Like we could either:
* maintain `timestamp` as a number everywhere, and only convert to `Date` in the components that need it, e.g. for date computation or rendering (like is done here)
* change the field name in `submissionData` (maybe use `date`, or `datetime`) to indicate the field is a `Date`

I know TypeScript helps us with type, but having the same field name be of different types in different context is confusing for humans, not compilers 😅

Reply:
whoops. you are completely correct - i played around with the types initially, thinking what would be the best type for the dto and then where to perform the conversion. I agree we should keep to ms from epoch as far as possible! this is fixed in [7b0417a](https://github.com/opengovsg/FormSG/pull/5672/commits/7b0417a219cd4c5680252eb74b20e28705deca47)